src/app.ts

```diff
@@ -16,3 +16,5 @@ App.start(() => {
   App.initRoutes(Routes);
   App.initSocketServer(SocketServer);
 });
+
+module.exports = App;
```

Comment:
End of file line. 

Its nice sometimes to use prettier on autosave to always have a new line in end of files